docs: Update logos and press materials

richb-hanover opened the pull request:
A number of updates to the "branding and press materials" repo:

* Update all "wordmarks" to use the proper font - Inter, ExtraBold/800 weight. (Some images were using an older font that no longer matches our site.)
* Update logos to be exactly the size of the art work (without any extra white space around the logo)
* Create consistently-named files:
   * prql-logo.xxx for the logo alone
   * prql-wordmark.xxx for the logo plus wordmark
* Display License info for both PRQL and the logo
* Add some "press material" information for external writers
* Add a test HTML page to ensure that the logos didn't have extra white space. (In the event that the logos ever need to be adjusted.)

What could happen next? After this is merged, we might copy these files back into the main PRQL repo, perhaps under `/web`. We could then remove any separate copies of the artwork (for example, https://prql-lang.org/img/icon.svg), potentially using the wordmark SVG in the header, and then remove this repo.

max-sixty commented on README.md:
```diff
@@ -1,14 +1,66 @@
+That makes it easier to use these images in graphic layout,
+since it's easier to add padding to an image than to remove it.
+
+_Note: Earlier versions of this repo did not
```
I don't think we need to retain backward compat — what could we do to remove this note?

max-sixty reviewed (commented):
Excellent! Thank you @richb-hanover, this is much better

Is `logo/SVG/Logo.svg` correct? It looks like that has some different font?

max-sixty commented:
> What could happen next? After this is merged, we might copy these files back into the main PRQL repo, perhaps under /web. We could then remove any separate copies of the artwork (for example, https://prql-lang.org/img/icon.svg), potentially using the wordmark SVG in the header, and then remove this repo.

I'm very open-minded, and don't see a need to have more repos unless we need them.

But is it difficult to have them under a different license in the main repo? Or it's OK to have a minority of the repo under a different license? I'm not sure what norms & policies are around this.

(note that we do already have a couple of files under GPL in the repo that we've copied from elsewhere)

richb-hanover commented:
> Is `logo/SVG/Logo.svg` correct? It looks like that has some different font?

Good catch. I have fixed it (I changed the font character's object to a path so it 'looks right'). 

I had been keeping that around (with the wrong font) for backward compatibility. But...

Are there any files in any of the PRQL repos that refer to this repo? If not, we can definitely delete files whose names don't conform to the new naming scheme.

re: Licenses. I don't know what policy we want to adopt (or what the convention might be...)

Let's merge these changes and work out the questions (other repo's pointing to this one; licensing) soon.

max-sixty commented:
> Are there any files in any of the PRQL repos that refer to this repo? If not, we can definitely delete files whose names don't conform to the new naming scheme.

No! Delete away!
